src/core/mount.js

```diff
@@ -7,6 +7,10 @@ import { persistor, store } from "./store";
 import ErrorBoundaryAlert from "../components/error-boundary-alert/ErrorBoundaryAlert";
 import { closeLastModal } from "./modal.slice";
 
+// Todo: This should be updated because React.createElement() is deprecated.
+// After the update, ensure that prefixes (identifierPrefix) are specified for all apps.
+// This will guarantee unique IDs everywhere useID() is utilized.
+
```

Comment:
Please move and reconsider this.

1. This is in direct relation to the `render()` call. Please move the comment directly above this call. Otherwise chances are that it will be overlooked
2. It is `render()` that is deprecated - not `createElement()`